frontends/main/next.config.js

```diff
@@ -44,6 +44,8 @@ const nextConfig = {
     return config
   },
 
+  productionBrowserSourceMaps: true,
```

Comment:
> During production builds, they are disabled to prevent you leaking your source on the client, unless you specifically opt-in with the configuration flag.

From [docs](https://nextjs.org/docs/app/api-reference/next-config-js/productionBrowserSourceMaps)

Seems like silly obfuscation to me.

Anyway, our source is public. I *think* keeping the source maps public will mean we don't need to upload them to sentry, but we'll only know for sure after we deploy it.